### PR TITLE
Add wgNoFollowLinks and wgNoFollowNsExceptions to ManageWiki

### PR DESCRIPTION
**Pull request by Kelvs599**

### CODE REVIEW

File: ManageWikiNamespaces.php [hunk 1]
```diff
@@ -58,6 +58,16 @@
 		'overridedefault' => false,
 		'requires' => [],
 	],
+	'wgNoFollowNsExceptions' => [
```
Comment (Universal-Omega) [inline ManageWikiNamespaces.php:61]: Needs to be added to LocalSettings.php.

**Review by Universal-Omega — CHANGES_REQUESTED** [review]

File: ManageWikiNamespaces.php (hunk 1 above)
Comment (Kelvs599) [reply ManageWikiNamespaces.php:61]: Looking through LocalSettings.php, it doesn't seem like any of the ManageWikiNamespaces configs are there?

Comment (Universal-Omega) [reply ManageWikiNamespaces.php:61]: Can you please still add this there? That's something that I'll look into later. It can't hurt to add it.

Comment (Kelvs599) [reply ManageWikiNamespaces.php:61]: Done.

**Pull request merged**